utils/validations/test_validate_simple_json.go

```diff
@@ -0,0 +1,212 @@
+	assert.GreaterOrEqual(t, notApplicableResults, params.NotApplicable, "Expected at least %d not applicable vulnerabilities in scan responses, but got %d not applicable vulnerabilities.", params.NotApplicable, notApplicableResults)
+
+	assert.GreaterOrEqual(t, len(results.SecurityViolations), params.SecurityViolations, "Expected at least %d security violations in scan responses, but got %d security violations.", params.SecurityViolations, len(results.SecurityViolations))
+	assert.GreaterOrEqual(t, len(results.LicensesViolations), params.LicenseViolations, "Expected at least %d license violations in scan responses, but got %d license violations.", params.LicenseViolations, len(results.LicensesViolations))
+	assert.GreaterOrEqual(t, len(results.OperationalRiskViolations), params.OperationalViolations, "Expected at least %d operational risk violations in scan responses, but got %d operational risk violations.", params.OperationalViolations, len(results.OperationalRiskViolations))
+
+	assert.GreaterOrEqual(t, len(results.Licenses), params.Licenses, "Expected at least %d Licenses in scan responses, but got %d Licenses.", params.Licenses, len(results.Licenses))
+}
+
+func ValidateSimpleJsonResults(t *testing.T, exactMatch bool, expected, actual formats.SimpleJsonResults) {
+	validateContent(t, exactMatch, StringValidation{Expected: expected.MultiScanId, Actual: actual.MultiScanId, Msg: "MultiScanId mismatch"})
+	validateContent(t, false, NumberValidation[int]{Expected: len(expected.Errors), Actual: len(actual.Errors), Msg: "Errors count mismatch"})
+	// Validate vulnerabilities
+	for _, expectedVulnerability := range expected.Vulnerabilities {
+		vulnerability := getVulnerabilityOrViolationByIssueId(expectedVulnerability.IssueId, actual.Vulnerabilities)
+		if !assert.NotNil(t, vulnerability, fmt.Sprintf("IssueId %s not found in the vulnerabilities", expectedVulnerability.IssueId)) {
+			return
+		}
+		validateVulnerabilityOrViolationRow(t, exactMatch, expectedVulnerability, *vulnerability)
+	}
+	// Validate securityViolations
+	for _, expectedViolation := range expected.SecurityViolations {
+		violation := getVulnerabilityOrViolationByIssueId(expectedViolation.IssueId, actual.SecurityViolations)
+		if !assert.NotNil(t, violation, fmt.Sprintf("IssueId %s not found in the securityViolations", expectedViolation.IssueId)) {
+			return
+		}
+		validateVulnerabilityOrViolationRow(t, exactMatch, expectedViolation, *violation)
+	}
+}
+
+func getVulnerabilityOrViolationByIssueId(issueId string, content []formats.VulnerabilityOrViolationRow) *formats.VulnerabilityOrViolationRow {
+	for _, result := range content {
+		if result.IssueId == issueId {
+			return &result
+		}
+	}
+	return nil
+}
+
+func validateVulnerabilityOrViolationRow(t *testing.T, exactMatch bool, expected, actual formats.VulnerabilityOrViolationRow) {
+	validateContent(t, exactMatch,
```

Comment:
returned value is ignored

Reply:
can be ignored if not used, just like you are ignoring the bool value in `assert` in most cases